lxd/patches.go

```diff
@@ -1487,4 +1488,33 @@ func patchOIDCGroupsClaimScope(_ string, d *Daemon) error {
 	return nil
 }
 
+// Remove shared.VarPath("backups") and shared.VarPath("images") symlinks.
+func patchRemoveBackupsImagesSymlinks(_ string, d *Daemon) error {
+	dirs := []string{
+		shared.VarPath("backups"),
+		shared.VarPath("images"),
+	}
+
+	for _, dir := range dirs {
+		if !shared.PathExists(dir) {
```

Comment:
do we need this extra syscall, can we instead detect the not exist error from `os.Lstat()` instead?